eo-maven-plugin/src/test/java/org/eolang/maven/ContainsFiles.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2016-2023 Objectionary.com
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.eolang.maven;
+
+import java.nio.file.Path;
+import java.util.Arrays;
+import org.cactoos.iterable.Mapped;
+import org.hamcrest.Description;
+import org.hamcrest.TypeSafeMatcher;
+
+/**
+ * Asserting that path contains a files matching provided globs.
+ * @since 0.31.0
+ */
+@SuppressWarnings({"JTCOP.RuleAllTestsHaveProductionClass", "JTCOP.RuleCorrectTestName"})
+final class ContainsFiles extends TypeSafeMatcher<Path> {
```

Comment:
@volodya-lombrozo I believe if we need extra functionality we shouldn't change existed classes but should create new ones. In my opinion this is how [OCP](https://en.wikipedia.org/wiki/Open%E2%80%93closed_principle) should work.

Reply:
@maxonfjvipon It's a good point. Although let's do a mental experiment:
1. After this change we will be able to use `ContainsFiles` instead of `ContainsFile`. Am I right? Since we can pass varargs into constructor, we can use `new ContainsFiles("**/*")` instead of `new ContainsFile("**/*")` **everywhere**.
2. So, the next step will be exactly that - we will just remove `ContainsFile` class (inline it into `ContainsFiles`) because we can do it, since we don't use `ContainsFile` anymore and the code will become smaller and simpler without redundant class. Profit.
3. We will have `ContainsFiles` instead of `ContainFile`.

I suggest to shortcut all this steps and just modify `ContainsFile`

Reply:
@maxonfjvipon It's a good point. Although let's do a mental experiment:
1. After this change we will be able to use `ContainsFiles` instead of `ContainsFile`. Am I right? Since we can pass varargs into constructor, we can use `new ContainsFiles("**/*")` instead of `new ContainsFile("**/*")` **everywhere**.
2. So, the next step will be exactly that - we will just remove `ContainsFile` class (inline it into `ContainsFiles`) because we can do it, since we don't use `ContainsFile` anymore and the code will become smaller and simpler without redundant class. Profit.
3. We will have `ContainsFiles` instead of `ContainFile`.

I suggest to shortcut all this steps and just modify `ContainsFile`